test_gtest/ParenthesesTests.cpp

```diff
@@ -14,3 +14,196 @@ TEST(ParenthesesTests, givenEmptyStringWhenIsBalancedIsCalledThenResultIsTrue)
     // THEN
     ASSERT_TRUE(result);
 }
+
+TEST(ParenthesesTests, givenRoundParenthesesWhenIsBalancedAlgoIsCalledThenResultIsTrue)
+{
+    //GIVEN
+    std::string input {"()(())()"};
+    Parentheses p;
```

Comment:
Obiekt Parentheses p jest używany we wszystkich  testach. Zamieńcie TEST na TEST_F, utwórzcie klasę ParenthesesTests i tam go wrzućcie. Dzięki temu możecie z każdego testu usunąć tę 1 linijkę. Wszystko co jest wspólne może w tej klasie wylądować.